clojure_liberator/src/clojure_liberator/data_access.clj

```diff
@@ -0,0 +1,22 @@
+(ns clojure-liberator.data-access)
+(use 'korma.db)
+(use 'korma.core)
+(require '[clojure.string :as str])
+
+(defdb eboshi (mysql {:db "eboshi_test"
+                       :user "root"
+                       :password ""
+                       ;; optional keys
+                       }))
+
+(declare clients users)
+
+(defentity clients)
+
+(defentity users)
```

Comment:
Just curious about this. Does `korma` require you to declare entities for each table in the db?